src/rest/connector/libs/apic/implementation.py

```diff
@@ -102,7 +102,9 @@ def connect(self, timeout=30, retries=3, retry_wait=10):
         if 'host' in self.connection_info:
             ip = self.connection_info['host']
         else:
-            ip = self.connection_info['ip'].exploded
+            ip = self.connection_info['ip']
+            if hasattr(ip, 'exploded'):
+                ip = ip.exploded
```

Comment:
To use an address that is not an IPAddress object, you can pass the `host` key instead.

For this change I suggest the following code:
```python
ip = self.connection_info['ip']
if isinstance(ip, (ipaddress.IPv4Address, ipaddress.IPv6Address)):
   ip = ip.exploded
```